tests/test_application.coffee

```diff
@@ -6,16 +6,24 @@ via the new configuration api introduced in space:base.
 
 class @CustomerApp extends Space.Application
 
+  @publish -> 'CustomerApp'
+
   RequiredModules: ['Space.eventSourcing']
 
   Dependencies: {
     commandBus: 'Space.messaging.CommandBus'
     eventBus: 'Space.messaging.EventBus'
-    Mongo: 'Mongo'
+    mongo: 'Mongo'
   }
 
   Configuration: {
-    useSnapshotting: true
+    testMode: true
+    eventSourcing: {
+      snapshotting: {
+        frequency: 2
+        collectionName: 'my_collection'
```

Comment:
I think we should remove this and let the default apply, then rig it in a specific test to check that functionality.